scripts/db_migrator.py

```diff
@@ -1122,6 +1124,27 @@ def migrate(self):
             version = next_version
         # Perform common migration ops
         self.common_migration_ops()
+        if os.environ["UTILITIES_UNIT_TESTING"] == "2":
```

Comment:
>UTILITIES_UNIT_TESTING [](http://example.com/codeflow?start=23&length=22)

I am thinking this detection is useful in production runtime as background task, not just unit test. It will help detect bugs like:
1. db_migrator generates a new config but not yang validated
2. load_minigraph generates a new config but not yang validated

Not sure if there are other places the yang validation could help in runtime. But let's only detect and generate CRIT syslog, not to block the existing workflow.

Reply:
https://github.com/sonic-net/sonic-buildimage/blob/master/src/sonic-py-common/sonic_py_common/logger.py#L101-L114
I can't find CRIT in logger.py

Reply:
Updated